src/mqt/predictor/evaluation.py

```diff
@@ -161,7 +161,7 @@ def evaluate_GHZ_circuits() -> None:
     )
     res_csv.append(list(results[0].keys()))
     for res in results:
-        res_csv.append(list(res.values()))
+        res_csv.append(list(res.values()))  # noqa: PERF401
```

Comment:
You forgot to replace this instance like the one above ;o)

Reply:
Oh. Now it is fixed there as well.